### ISSUE TITLE
BatchNorm 与 LayerNorm | 各有未来

### ISSUE THREAD

**Issue by alyssaasa** [alyssaasa/alyssaasa.github.io#28]
https://alyssaasa.github.io/2019/11/08/LayerNorm-%E4%B8%8E-BatchNorm/ 

 🌼  Batch Normalization1. 作用：batch normalization 主要起到正则化的作用，可以加速网络收敛，并防止梯度消失。具体解释可以看： https://www.bilibili.com/video/av16000304
